tiledb/sm/storage_manager/storage_manager_canonical.h

```diff
@@ -77,9 +77,7 @@ class GroupDetails;
 class Metadata;
 class MemoryTracker;
 class Query;
-class QueryCondition;
 class RestClient;
-class UpdateValue;
 class VFS;
```

Comment:
Not directly related to this PR., but I believe the forward declaration `class VFS` could also be removed in the currently-open PR for removing `StorageManager::vfs()`. If not in that one, remember to take it out adventitiously soon in some other related PR.

Reply:
Will do so in my next removal of `compute_tp`, thanks.